assembl/static2/js/app/components/home/video.jsx

```diff
@@ -3,6 +3,8 @@ import { connect } from 'react-redux';
 import { Translate } from 'react-redux-i18n';
 import { Grid, Row, Col } from 'react-bootstrap';
 
+import Medias from '../common/medias';
+
 class Video extends React.Component {
```

Comment:
// @flow :)

Reply:
Same here. It's an old file habibi. Not in the scope

Reply:
We said we try to add flow on 10 files per dev, there is no ticket for that, it's just something you do when you touch a file where it's missing. At least I would take note that it needs to be done on video.jsx

Reply:
I'm gonna do it now ;) I have time...